exo-run/features/steps/steps-then.ls

```diff
@@ -47,13 +52,12 @@ module.exports = ->
 
 
   @Then /^my machine is running ExoCom$/ timeout: 10_000, (done) ->
-    @process.wait /exocom  ExoCom WebSocket listener online at port/, done
+    @process.wait /ExoCom WebSocket listener online at port/, done
 
 
-  @Then /^my machine is running the services:$/ timeout: 10_000, (table, done) ->
-    async.each [row['NAME'] for row in table.hashes!],
-               ((name, cb) ~> @process.wait "'#{name.to-lower-case!}' is running", cb),
-               done
+  @Then /^my machine is running the services:$/ (table) ->
+    for row in table.hashes!
+      expect(child_process.exec-sync('docker ps --format {{.Names}}/{{.Status}}') |> (.to-string!) |> (.split os.EOL) |> any (.includes "#{row.NAME}/Up")).to.be.true
```

Comment:
👍 We are getting rid of the old `DockerHelper` command line methods and switching to `dockerode`. I'm working on a PR that does this, so this line will eventually reflect that.